### DISCUSSION TITLE
Custom Rulesets In Base Game

### DISCUSSION THREAD

**Discussion by SealsarecuteR** [ppy/osu#18267] [Ideas] [answered]
This one kinda goes with my other idea, but having some of the custom rulesets in the base game would be cool. Some of the rulesets are buggy like Osu!Diva or don't work due to updates to Lazer like Soyokaze, so having them update with the other 4 could solve that issue.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/16937#discussioncomment-2217859
